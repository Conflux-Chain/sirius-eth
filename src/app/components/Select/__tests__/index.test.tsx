import React from 'react';
import { render } from '@testing-library/react';
import { Select } from '..';

const renderComponent = () => render(<Select />);
describe('<Select />', () => {
  it('should match snapshot', () => {
    const component = renderComponent();
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
