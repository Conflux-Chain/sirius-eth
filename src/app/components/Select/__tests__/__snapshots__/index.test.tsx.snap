// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> should match snapshot 1`] = `
.c0 .select.sirius-select {
  height: 2.2857rem;
}

<div
  class="c0"
>
  <div
    class="jsx-1615005387 select  line sirius-select"
  >
    <span
      class="jsx-1615005387 value placeholder"
    >
      <span
        class="jsx-2608415601"
      />
    </span>
    
    
    <div
      class="jsx-1615005387 icon"
    >
      <svg
        class="jsx-3859518410"
        fill="none"
        height="1.25em"
        shape-rendering="geometricPrecision"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1"
        viewBox="0 0 24 24"
        width="1.25em"
      >
        <path
          class="jsx-3859518410"
          d="M6 9l6 6 6-6"
        />
      </svg>
    </div>
  </div>
</div>
`;
